docs/product-highlights-and-benefits/live-context.mdx

```diff
@@ -59,33 +65,42 @@ You can also manually update this by adding and enabling Pieces OS in the follow
 #### Privacy & Security > Screen and System Audio Recording
 ![Screen and System Audio Recording Permissions](/wpe/screen-and-system-audio-permissions.png)
 
-## Recommendations and Best Practices
+## Live Context Use Cases
 
 :::tip
 We recommend clicking around your workflow between chats, websites, emails, code, etc., while the WPE is running before asking a question to get the most value out of it (i.e., data capture).
 :::
 
-### Interaction with Language Models (LLM)
-
-- **Cloud LLM:** If you are using a cloud-based LLM, the data identified as relevant is sent to the cloud LLM for processing.
-- **Local LLM:** If you are using a local LLM, the data remains on your device, ensuring that all processing happens locally without any data leaving your device.
+## Example Prompts
```

Comment:
Should this just be `Best Practices` since we removed the `Use Cases` H2 and now prompting, text practices, and llm interaction falls under `Example Prompts`?

Reply:
yes!